src/App.js

```diff
@@ -21,6 +22,7 @@ function App() {
             <Route path="/catPresentation" exact>
               <CatPresentationPage />
             </Route>
+            <Route path="/:id" children={<CatDetailsPage />}></Route>
```

Comment:
```suggestion
            <Route path="/:id">
                <CatDetailsPage />
            </Route>
```

You can always pass `children` prop inside the component, like here:

```tsx
            <Route path="/catPresentation" exact>
              <CatPresentationPage />
            </Route>
```